src/utils/drawScreen.ts

```diff
@@ -1,7 +1,7 @@
-import readline from "readline"
+import config from "@config"
```

Comment:
what the hell does the @ button do? are you trying to send me an email?

Reply:
Nobody does this like that. Alias `~` to the project root and write `~/config.ts`, `~/utils`, etc.

Reply:
@vezaynk It's a great point as the '@' is often used as a prefix when importing certain npm modules. I've pushed the changes up.